_includes/author-profile.html

```diff
@@ -5,19 +5,10 @@
 
   {% if author.avatar %}
     <div class="author__avatar">
-      {% if author.avatar contains "://" %}
-        {% assign author_src = author.avatar %}
-      {% else %}
-        {% assign author_src = author.avatar | relative_url %}
-      {% endif %}
+			{% assign author_src = author.avatar | relative_url %}
```

Comment:
Can further simplify by removing this `assign` and add the `relative_url` filter directly to `author.avatar` in the image elements.

```html
<img src="{{ author.avatar | relative_url }}" alt="{{ author.name }}" itemprop="image">
```